cosmoz-image-viewer.js

```diff
@@ -392,6 +392,14 @@
 
 		_onResize() {
 			this.set('_imageContainerHeight', this._scroller.scrollHeight);
+			this.debounce('hideAbsoluteContentOnSmallHeight', () => {
+				const absElements = Polymer.dom(this.root).querySelectorAll('.hide-on-small-height');
+				if (this.offsetHeight < 100) {
+					absElements.forEach(el => el.setAttribute('hidden', true));
```

Comment:
I think you should rather use a `hidden` attribute on elements you want to hide, bound to a property whose value is true/false depending of height.
For page number, you can additionnaly add this property to the computed binding of `_showPageNumber`, for example:
```
_showPageNumber: {
   type: Boolean,
   computed: '_computeShowActions(showPageNumber, images.length, "pagenumber")'
}
```